Dockerfile

```diff
@@ -48,3 +49,5 @@ COPY --from=builder /usr/src/app/build ./build
 COPY --from=builder /usr/src/app/api.yaml ./api.yaml
 EXPOSE 5000
 CMD node build/server/bundle.js
+HEALTHCHECK --interval=2m --timeout=10s --start-period=1m --retries=3 \
```

Comment:
```suggestion
# See https://docs.docker.com/engine/reference/builder/#healthcheck
HEALTHCHECK --interval=2m --timeout=10s --start-period=1m --retries=3 \
```

Reply:
Not agree. Why we add link about that directive and not add to others? Everyone who ever create Dockerfile know where to look for help about Dockerfile directives.